deb/gen-deb-ver

```diff
@@ -8,11 +8,6 @@ SUFFIX=${SUFFIX:=ce}
 
 [[ $# < 2 ]] && echo 'not enough args' && exit 1
 
-DATE_COMMAND="date"
-if [[ $(uname) -eq "Darwin" ]]; then
-    DATE_COMMAND="docker run --rm alpine date"
```

Comment:
Looks like this was because macOS `date` doesn't have `--date="string"`?

I replaced this for `echo "datestring" | xargs date -d ....`

Reply:
Argh; looks like that doesn't work on macOS after all 😕